Ehr_OOP.py

```diff
@@ -0,0 +1,97 @@
+"""Bio stats assignment #OOP Mohammad Anas."""
+
+
+from datetime import datetime
+
+
+class Patient:
+    """Store a patient data."""
+
+    def __init__(self, PatientID):
+        """Instantiate Patient Class."""
+        self.patientID = PatientID
+        self.gender = None
+        self.DOB = None
+        self.race = None
+
+    def __repr__(self):
```

Comment:
`__repr__` typically produces a detailed string representation. For something this simple, I suspect that you want to overload `__str__` instead.